cobigen/cobigen-textmerger/src/test/java/com/devonfw/cobigen/textmerger/TextAppenderTest.java

```diff
@@ -32,7 +32,8 @@
     public void testMerge_appendWithoutNewLineNoAnchors() throws Exception {
         TextAppender appender = new TextAppender("textmerge_append", false);
         String mergedString = appender.merge(new File(testFileRootPath + "BaseFile.txt"), "Test3", "UTF-8");
-        assertThat(mergedString).isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile.txt")));
+        mergedString = mergedString.replaceAll("\\n|\\r", "");
+        assertThat(mergedString).isEqualTo(FileUtils.readFileToString(new File(testFileRootPath + "MergedFile.txt")).replaceAll("\\n|\\r", ""));
```

Comment:
Shouldn't you use the regex `"\\r\\n|\\n|\\r"` that you have used in other files?